### PR TITLE
feat(Oneanime): add new presence

### PR DESCRIPTION
**Pull request by KanekiWeb**
## Description 
Add presence for [oneanime.fr](oneanime.fr)

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

  

  

  

  

  

  

</details>

### CODE REVIEW

**Review by theusaf — CHANGES_REQUESTED** [review]
Your file structure is incorrect. `dist/metadata.json` should be just `metadata.json`. Also, `tsconfig.json` should not be committed.

File: websites/O/Oneanime/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"description": {
+		"en": "Take advantage of our new OneAnime platform to watch your favorite anime totally free, in VF and VOSTFR!",
+		"fr": "Profiter de notre nouvelle plateforme OneAnime pour regarder vos animés préféré totalement gratuitement, en VF et VOSTFR!"
+	},
+	"url": [
+		"oneanime.fr",
+		"www.oneanime.fr"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/NLrk8Qp.png",
```
Comment (theusaf) [inline websites/O/Oneanime/dist/metadata.json:17]: Logos should be at least 512x512px. Find a larger version or upscale it using a tool like waifu2x.

File: websites/O/Oneanime/presence.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+presence.on(
+	"iFrameData",
+	(data: { current: number; duration: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
```
Comment (theusaf) [inline websites/O/Oneanime/presence.ts:19]: Imgur links are preferred over Discord asset names.
This applies to `largeImageKey` and `smallImageKey`

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
"Files changed 0"

**Comment by KanekiWeb** [comment]
I have make a 2nd pull request

**Pull request closed without merging**